packages/editor/src/hooks/custom-sources-backwards-compatibility.js

```diff
@@ -15,7 +16,9 @@ function useMetaAttributeSource( name, _attributes, _setAttributes ) {
 
 	if ( Object.values( attributeTypes ).some( ( type ) => type.source === 'meta' ) ) {
 		// eslint-disable-next-line react-hooks/rules-of-hooks
-		const [ meta, setMeta ] = useEntityProp( 'postType', 'post', 'meta' );
+		const { type } = useSelect( ( select ) => select( 'core/editor' ).getCurrentPost(), [] );
```

Comment:
`getCurrentPostType` selector maybe?

Reply:
Sure, we'll need to change it later, but it's better for performance now.